dev/devicelab/README.md

```diff
@@ -7,15 +7,79 @@ This package contains the code for test framework and the tests. More generally
-(There is currently no public view of this data, unfortunately.)
+Build results are available at https://flutter-dashboard.appspot.com.
 
-# Prerequisites
+# Reading the dashboard
 
-You must set the `ANDROID_HOME` environment variable to run tests on Android. If
-you have a local build of the Flutter engine, then you have a copy of the
-Android SDK at `.../engine/src/third_party/android_tools/sdk`.
+## The build page
+
+The build page is accessible at https://flutter-dashboard.appspot.com/build.html.
+This page reports the health of build servers, called _agents_, and the statuses
+of build tasks.
+
+### Agents
+
+A green agent is considered healthy and ready to receive new tasks to build. A
+red agent is broken and does not receive new tasks.
+
+In the example below, the dashboard shows that the `linux2` agent is broken and
+requires attention. All other agents are healthy.
+
+![Agent statuses](images/agent-statuses.png)
+
+### Tasks
+
+The table below the agent statuses displays the statuses of build tasks. Task
+statuses are color-coded. The following statuses are available:
+
+**New task** (light blue): the task is waiting for an agent to pick it up start
+the build.
+
+**Task is running** (spinning blue): an agent is currently building the task.
+
+**Task succeeded** (green): an agent reported a successful completion of the
+task.
+
+**Task is flaky** (yellow): the task was attempted multiple time, but only the
+latest attempt succeeded (we currently only try twice).
+
+**Task failed** (red): the task failed all of the attempts.
+
+**Task underperformed** (orange): currently not used.
+
+**Task was skipped** (transparent): the task is not scheduled for a build. This
+usually happens when a task is removed from `manifest.yaml` file.
+
+**Task status unknown** (purple): currently not used.
+
+In addition to color-coding, a task may display a question mark. This means
+that the task was marked as flaky manually. The status of such task is ignored
+when considering whether the build is broken or not. For example, if a flaky
+task fails, GitHub will not prevent PR submissions. However, if the latest
+status of a non-flaky task is red, all pending PRs will contain a warning about
+the broken build and recommend caution when submitting.
+
+Legend:
+
+![Task status legend](images/legend.png)
+
+The example below shows that commit `e122d5d` caused a wide-spread breakage,
+which was fixed by `bdc6f10`. It also shows that Travis, AppVeyor and Chrome
+Infra (left-most tasks) decided to skip building these commits. Hovering over
+a cell will pop up a tooltip containing the name of the broken task. Clicking
+on the cell will open the log file in a new browser tab (only visible to core
+contributors as of today).
+
+![Broken Test](images/broken-test.png)
+
+## Why is a task stuck on "new task" status?
+
+The dashboard aggregates build results from multiple build environments,
```

Comment:
I presume it's not easily possible to tell if a task is being skipped rather than just hasn't started yet?

Reply:
If it's skipped intentionally it will have no color at all; the table cell will look as if missing. If the task is _meant_ to run but is skipped due to some infrastructure peculiarities then yeah, it will simply remain as "New".